src/pages/contact.tsx

```diff
@@ -7,7 +7,97 @@ const ContactPage: React.FC<PageProps> = () => {
   return (
     <Layout>
       <SEO />
-      {/** TODO */}
+      <div className="site-section ftco-subscribe-1 site-blocks-cover pb-4" style={{ backgroundImage: `url('/images/bg_3.jpg')` }}>
+        <div className="container">
+          <div className="row align-items-end">
+            <div className="col-lg-7">
+              <h2 className="mb-0">Kapcsolat</h2>
+              <p>Elérhetőség, térkép és fényképek</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="custom-breadcrumbs border-bottom">
+        <div className="container">
+          <Link to="/">Kezdőlap</Link>
+          <span className="mx-3 icon-keyboard_arrow_right" />
```

Comment:
on Firefox ( windows ) `icon-keyboard_arrow_right` doesn't seem to work.

Reply:
Yep these are silly little design problems that I will repair in an other PR but right now the content was priority not the design